shoop/core/fields/tagged_json.py

```diff
@@ -5,6 +5,20 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 from __future__ import unicode_literals
+
+import datetime
+import decimal
+from enum import Enum
+
+import django.utils.dateparse as dateparse
+from django.core.exceptions import ImproperlyConfigured
+from django.utils import six
+from jsonfield.encoder import JSONEncoder
+from six import text_type
+
+from shoop.utils.importing import load
+from shoop.utils.iterables import first
+
 """
 "Tagged JSON" encoder/decoder.
```

Comment:
This is interesting... If the imports go before the module docstring, it won't be treated as module docstring at all. Also the type of module docstring in Python 2 is unicode even when the unicode_literals import follows the docstring.
```
$ cat a.py 
"""
Docstring.
"""

from __future__ import unicode_literals
import sys

print(sys.version, type(__doc__))
$ cat b.py 
from __future__ import unicode_literals
import sys

"""
Docstring.
"""

print(sys.version, type(__doc__))
$ python a.py
('2.7.6 (default, Jun 22 2015, 17:58:13) \n[GCC 4.8.2]', <type 'unicode'>)
$ python b.py
('2.7.6 (default, Jun 22 2015, 17:58:13) \n[GCC 4.8.2]', <type 'NoneType'>)
$ python3 a.py
3.4.0 (default, Jun 19 2015, 14:20:21) 
[GCC 4.8.2] <class 'str'>
$ python3 b.py
3.4.0 (default, Jun 19 2015, 14:20:21) 
[GCC 4.8.2] <class 'NoneType'>
```